LoTrac_target.pl

```diff
@@ -172,7 +165,7 @@ sub fasta_seq_length {
     my ($seq) = @_;
     #open ( my $q_seq, "<", $seq ) or die "Could not open file '$seq': $!";    
     my @lines = split /\n/, $seq;
-    my $final_line;
+    my $final_line = "";
```

Comment:
This prevents a perl warning (uninitialized variable or something along those lines) as the first time the variable is concatenated to, it is essentially undefined.